src/lib/features/project-status/projects-status.e2e.test.ts

```diff
@@ -0,0 +1,40 @@
+import dbInit, { type ITestDb } from '../../../test/e2e/helpers/database-init';
+import {
+    type IUnleashTest,
+    setupAppWithCustomConfig,
+} from '../../../test/e2e/helpers/test-helper';
+import getLogger from '../../../test/fixtures/no-logger';
+
+let app: IUnleashTest;
+let db: ITestDb;
+
+beforeAll(async () => {
+    db = await dbInit('projects_status', getLogger);
+    app = await setupAppWithCustomConfig(
+        db.stores,
+        {
+            experimental: {
+                flags: {
+                    strictSchemaValidation: true,
+                },
```

Comment:
Do we need to set this flag manually still? I thought it was auto-on? 🤔

Reply:
This is not auto on, we should enable it, but I think our schemas still suck enough that we cant :)